src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -412,8 +412,10 @@ private function registerDoctrineExtensionConfiguration(ContainerBuilder $contai
             return;
         }
 
-        $container->removeDefinition('api_platform.doctrine.orm.query_extension.eager_loading');
-        $container->removeDefinition('api_platform.doctrine.orm.query_extension.filter_eager_loading');
+        $container->removeAlias('api_platform.doctrine.orm.query_extension.eager_loading');
```

Comment:
It should be the inverse IIUC.

Reply:
:+1: this is because of https://github.com/api-platform/core/pull/1557 !

You should remove the alias `EagerLoadingExtension::class` (please use this instead of the string) and remove the definition by it's id indeed.

Nice catch!

Reply:
You are right, I misread the error message, the problem is on the alias. I'm fixing it right now